impose/flblend.py

```diff
@@ -95,19 +95,24 @@ def blend_rgb(self):
         stack_r = np.zeros((shape[0], shape[1], len(self)), dtype=float)
         stack_g = np.zeros((shape[0], shape[1], len(self)), dtype=float)
         stack_b = np.zeros((shape[0], shape[1], len(self)), dtype=float)
+        stack_v = np.zeros((shape[0], shape[1], len(self)), dtype=float)
```

Comment:
Could you please add a comment here what `stack_v` is used for (also mention that you did it because of #25), so the reader knows what happens?

Reply:
Done.